samples/csharp/web/avatar/README.md

```diff
@@ -0,0 +1,103 @@
+# Instructions to run Microsoft Azure TTS Talking Avatar sample code
+
+## Pre-requisites
+
+* Follow [Text to speech quickstart](https://learn.microsoft.com/azure/ai-services/speech-service/get-started-text-to-speech?pivots=programming-language-python#set-up-the-environment) to set up the environment for running Speech SDK in python.
+
+## Basic Sample
+
+This sample demonstrates the basic usage of Azure text-to-speech avatar real-time API.
+
+* Step 1: Open a console and navigate to the folder containing this README.md document.
+    * Set below varibles in appsettings.json:
+        * `SpeechRegion` - the region of your Azure speech resource, e.g. westus2.
+        * `SpeechKey` - the subscription key of your Azure speech resource.
```

Comment:
Please rename subscription key to API key

Reply:
updated